### ISSUE TITLE
Question: Other Dokis Opinion Topic

### ISSUE THREAD

**Issue by scohn** [Monika-After-Story/MonikaModDev#3148]
Is it possible to repeat the new topic about choosing your second favorite doki? I kinda wanna see what all the options lead to.

**Comment by multimokia** [comment]
Absolutely, simply follow this path: `Repeat Conversation>Club Members>[Your Names]'s second choice`

**Comment by Rai99** [comment]
The option you choose currently doesn't affect anything else, but there was some discussion about overhauling that topic to add a bit more depth.
